src/openrct2/world/Footpath.h

```diff
@@ -37,60 +37,12 @@ enum class RailingEntrySupportType : uint8_t
     Count
 };
 
-enum
-{
-    FOOTPATH_ENTRY_FLAG_SHOW_ONLY_IN_SCENARIO_EDITOR = (1 << 2),
-    FOOTPATH_ENTRY_FLAG_IS_QUEUE = (1 << 3),
-    FOOTPATH_ENTRY_FLAG_NO_SLOPE_RAILINGS = (1 << 4),
-};
-
-#pragma pack(push, 1)
-struct FootpathEntry
-{
-    StringId string_idx;                  // 0x00
-    uint32_t image;                       // 0x02
-    uint32_t bridge_image;                // 0x06
-    RailingEntrySupportType support_type; // 0x0A
-    uint8_t flags;                        // 0x0B
-    uint8_t scrolling_mode;               // 0x0C
-
-    constexpr uint32_t GetQueueImage() const
-    {
-        return image + 51;
-    }
-    constexpr uint32_t GetPreviewImage() const
-    {
-        return image + 71;
-    }
-    constexpr uint32_t GetQueuePreviewImage() const
-    {
-        // Editor-only paths usually lack queue images. In this case, use the main path image.
-        if (flags & FOOTPATH_ENTRY_FLAG_SHOW_ONLY_IN_SCENARIO_EDITOR)
-        {
-            return GetPreviewImage();
-        }
-
-        return image + 72;
-    }
-    constexpr uint32_t GetRailingsImage() const
-    {
-        return image + 73;
-    }
-};
-assert_struct_size(FootpathEntry, 13);
-#pragma pack(pop)
-
 struct PathSurfaceDescriptor
 {
     StringId Name;
     uint32_t Image;
     uint32_t PreviewImage;
     uint8_t Flags;
-
-    inline constexpr bool IsEditorOnly() const
```

Comment:
Interesting, this was unused?

Reply:
https://github.com/OpenRCT2/OpenRCT2/commit/4cc717ccdbece7d193a9c9e1e2ed1bd6e74e22bd introduced this, but its uses seem to be gone now.

Reply:
Found it: https://github.com/OpenRCT2/OpenRCT2/commit/4f319d6ed4cb4dc032e2625741195d1c3931ed19